### ISSUE TITLE
Window key Error: vtki PyQt Interface

### ISSUE THREAD

**Issue by dangit6413** [akaszynski/vtki#92]
Example PyQt5 vtki QtInteractor has an error.
When I push "window key", contents in the window disappear like below.

![image](https://user-images.githubusercontent.com/34636679/52842346-27645480-3142-11e9-934d-d37e43f8c42e.png)

**Comment by akaszynski** [comment]
The windows key right?

**Comment by banesullivan** [comment]
Does this happen with the `vtki.BackgroundPlotter` too? Here's a snippet to run one:

```py
import vtki
from vtki import examples

dataset = examples.load_uniform()

tool = vtki.OrthogonalSlicer(dataset)
```

**Comment by dangit6413** [comment]
@banesullivan 
I'm sorry but above code is not proper to check I guess.
It freezes after showing model and does not respond.

**Comment by banesullivan** [comment]
If you are not in an IPython environment, you'll need some sort of way to pause the thread. Two options:

1. Add a line to prompt the user: `input('foo:')` to pause the main thread.
2. Use an IPython interpreter. For example, from command line:

```bash
$ python
>>> import vtki
>>> from vtki import examples
>>> dataset = examples.load_uniform()
>>> p = vtki.BackgroundPlotter()
>>> p.add_mesh(dataset)
```

**Comment by akaszynski** [comment]
fixed in 3a2d08ecc0c28d74e48395e03913e7cfa29b4f8e.  It will be included in the next release.
